modules/500-upmeter/hooks/dynamic_probe/common_test.go

```diff
@@ -0,0 +1,29 @@
+/*
+Copyright 2021 Flant JSC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package dynamic_probe
```

Comment:
Why not dynamic_probes? Like in openapi.

Reply:
Because in Go, it is idiomatic to name packages in singular form. From stdlib: 

- context
- flag
- errgroup
- container
- database
- flag
- path
- regexp
- expvar
 
There are exceptions when the word collides with built-in type or reserved word: 

- bytes
- strings
- errors

What is not idiomatic, though, it is using the underscore.